src/bindings/xfconf/README.md

```diff
@@ -0,0 +1,103 @@
+
+This can be also achieved using the [system replace script](scripts/replace-system-xfconf.sh).
+These changes can be reverted with the [system restore script](scripts/restore-system-xfconf.sh).
+Note, that both scripts must be run directly within the build root.
+However, there are more convenient ways to achieve that as described [below](#using-the-binding-as-a-replacement-in-xfce).
+
+## Using the binding as a replacement in Xfce
+
+It is currently possible to use this binding instead of Xfconf in order to start and use Xfce.
+
+**Caution:** Although it is in general possible to use this binding as a replacement it is not recommended at all.
+Currently, it shows an unpredictable behavior for an unknown reason.
+A few of the undesired effects are:
+
+- Inconsistent layout of the taskbar every session restart
+- Missing symbols in menus despite being configured to appear
+- Weird animations during the session startup
+  One guess is, this is the result of race-conditions since Xfce consists of multiple components which concurrently read and write configuration.
+
+Use with caution and on non-production systems such as virtual machines only.
+
+The following instructions only have an effect on the current user which allows a quick revert through a different user.
+
+Build elektra as instructed in the [COMPILE](../../../doc/COMPILE.md#developer-options) and do not forget to include the `xfconfbinding`.
+
+After building, the library has to be overridden with the [user replace script](scripts/replace-user-xfconf.sh).
+Again, the working directory must be the build root.
+Then the Xfce configuration must be initialized using:
+
+1. `source $HOME/.xprofile`
+2. running the [population script](scripts/populate-xfconf.sh)
+
+When done, the graphical session can be started and will now use elektra for the Xfce configuration.
+
+To restore the usage of the systems Xfconf, it is enough to remove the `LD_*` exports from `$HOME/.xprofile`
+
+## Debugging
+
+For debugging purposes, it might be useful to output and log all debug messages from glib.
+This can be done by appending the `G_MESSAGES_DEBUG=all` environment variable to all the above-mentioned places.
+However, this is already done by the [user replace script](scripts/replace-user-xfconf.sh).
+All components responsible for starting the Xfce session will log their output wherever the display-manager stores the log files.
+All applications started from the command line will put their debug log to the stderr.
+
+## Quality
+
+Note that this binding only handles configuration directly handles within Xfconf.
+It is not able to configure settings which live outside Xfconf such as Gtk themes.
+However, since settings such as the Gtk theme part of the Xfce desktop itself, this binding is not able to configure Xfce fully.
```

Comment:
```suggestion
However, since settings such as the Gtk theme are part of the Xfce desktop itself, this binding is not able to configure all aspects of Xfce.
```

Reply:
Should be done by 6506d0e23efa25047381ef0cd2afa5aed13da3f7 or before